### ISSUE TITLE
Add AdaBoostRegressor

### ISSUE THREAD

**Issue by Thilakraj1998** [blobcity/autoai#60]
Add AdaBoostRegressor model into the library.

Primary File to Change: https://github.com/blobcity/autoai/blob/main/blobcity/config/regressor_config.py

Reference AdaBoostRegressor Implementation:
https://github.com/blobcity/ai-seed/blob/main/Regression/Adaptive%20Boosting/AdaBoostRegressor.ipynb

Official API Refer for parameters:
https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.AdaBoostRegressor.html

Dependencies if any, must be appropriately added. Test run of the train function on a regression problem must pass, and the function must attempt to train an AdaBoostRegressor as a potential best fit model.

**Comment by 26tanishabanik** [comment]
@Thilakraj1998  For the adaboost regressor, do I need to define the base estimator as an object, like this "ensemble.DecisionTreeClassifier(max_depth=3)"?
